director/stat_test.go

```diff
@@ -89,24 +91,32 @@ func TestQueryServersForObject(t *testing.T) {
 			Type:    server_structs.OriginType}
 		mockServerAdPrivateTopology := server_structs.ServerAd{
 			Name:         "originPrivateTopology",
-			URL:          url.URL{Host: "mock-origin-5.com", Scheme: "https"},
-			AuthURL:      url.URL{Host: "mock-origin-5-auth.com:8444", Scheme: "https"},
+			URL:          url.URL{Host: "mock-private-topo-origin.com", Scheme: "https"},
```

Comment:
This is probably a minor-enough detail that it shouldn't effect anything, but the `URL` field from Topology is always `http`, while the `AuthURL` field should always be `https`.